packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -751,7 +751,7 @@ def _serialize_body_parameter(self, builder: OperationType) -> List[str]:
         elif self.code_model.options["models_mode"] == "dpg":
             create_body_call = (
                 f"_{body_kwarg_name} = json.dumps({body_param.client_name}, "
-                "cls=AzureJSONEncoder)  # type: ignore"
+                "cls=AzureJSONEncoder, exclude_readonly=True,\n exclude_none=False)  # type: ignore"
```

Comment:
we `exclude_none` `True` right now, it's just that it's currently handled in the JSON encoder. I feel it'll be clearer if we just pass in `True` here

Reply:
I'm a little confused. Let me give an example:
```python
class TestModel(Model):
    prop1: str
    prop2: str
    prop3: str
model = TestModel({"prop1": None, "prop2": NULL})
json.dumps(model, cls=AzureJSONEncoder, exclude_none=True)  # the result should be {"porp2": null}
json.dumps(model, cls=AzureJSONEncoder, exclude_none=False)  # test result should be {"prop1": null, "prop2": null}
```